divisible.rb

```diff
@@ -0,0 +1,11 @@
+def array_sort
+    my_range = Array((1..100))
+      sorted_array = Array.new
```

Comment:
```suggestion
      filtered_array = Array.new
```

Sorting and filtering are two different concepts, so I should clarify this. `my_range`, as soon as you assigned a value, was already _sorted_ in ascending numerical order, but what your function does is actually _filter_ all the multiples of 2, 3 and 5 by selectively _removing_ certain elements. The order did not change (it's still numerically ascending), so it's not a sort. This is reflected in the Ruby documentation for [Array#sort](https://ruby-doc.org/core-3.0.1/Array.html#method-i-sort) and [Array#filter](https://ruby-doc.org/core-3.0.1/Array.html#method-i-filter). `#sort` should not change the number of elements in the array, but `#filter` is designed to do so.